en/docs/choreo-samples/explore-the-demo-organization.md

```diff
@@ -15,21 +15,17 @@ The demo organization in Choreo provides a hands-on view of a fully deployed sys
 1. Sign in to the [Choreo Console](https://console.choreo.dev/).
 2. In the Choreo Console header, click the **Organization** list to view available organizations.  
 3. Under **Invited Organizations**, you will see the **Demo Organization**.  
-4. To gain access to the **Demo Organization**, click **Join** corresponding to it.
+4. To gain access to the **Demo Organization**, click **Join** next to it.
 
-## Demo organization overview
+## Demo Organization Overview
 
 When you join the demo organization, you gain access to a comprehensive, read-only sample application for managing customer rewards. This application illustrates a complete use case, showcasing the capabilities of Choreo.
 
 Key aspects you can explore include:
 
- - Projects and components: Take a look at the sample projects and their respective components, where each component serves a distinct function within the customer rewards management system.
+- **Projects and Components**: View the sample projects and their respective components, where each component serves a distinct function within the customer rewards management system.
```

Comment:
Since we follow the Google style guide capitalization guidelines, we avoid using title caps for lists and such. https://developers.google.com/style/capitalization#capitalization-in-lists